src/background.js

```diff
@@ -21,15 +21,26 @@ function init() {
   browser.alarms.onAlarm.addListener(handleWake);
   browser.notifications.onClicked.addListener(handleNotificationClick);
   browser.runtime.onMessage.addListener(handleMessage);
-  handleWake();
+  browser.storage.local.get().then(items => {
+    const due = Object.entries(items).filter(item => item[1].time === 'next');
```

Comment:
Not sure if we should abstract the `'next'` constant into a separate file or something.
I think it's used in about 8 places in this PR (probably not that we're thinking of refactoring this ever, but still, thought I'd throw this out there).